lib/index.js

```diff
@@ -51,11 +56,15 @@ function lint(editor) {
     plugins.push(recommended);
   }
 
+  if (styleGuideWithoutConfig) {
+    plugins.push(styleGuide);
```

Comment:
🐛 The setting description implies that if enabled this will be the _only_ configuration used, but this code simply adds it to the list, which is correct?